Lib/_pyrepl/completing_reader.py

```diff
@@ -258,10 +258,14 @@ def after_command(self, cmd: Command) -> None:
     def calc_screen(self) -> list[str]:
         screen = super().calc_screen()
         if self.cmpltn_menu_vis:
-            ly = self.lxy[1]
+            ly = self.lxy[1] + 1
             screen[ly:ly] = self.cmpltn_menu
-            self.screeninfo[ly:ly] = [(0, [])]*len(self.cmpltn_menu)
-            self.cxy = self.cxy[0], self.cxy[1] + len(self.cmpltn_menu)
```

Comment:
This is the big win here, we no longer need to mess with the cursor when displaying the menu!